tests/ecdsa/mod.rs

```diff
@@ -72,6 +74,46 @@ fn round_trip_claim() {
     assert_eq!(my_claims, token_data.claims);
 }
 
+#[cfg(feature = "use_pem")]
+#[test]
+fn expiry_and_nbf() {
```

Comment:
make it explicit in the name it's testing the new feature

Reply:
I'm not sure what you mean by this.